### PR TITLE
Fixing BDD tags exception + Window explorer issue

### PR DESCRIPTION
**Pull request by EliranBarDavid**
Fixing WIndow Explorere issues

### CODE REVIEW

File: Ginger/Ginger/WindowExplorer/Common/WindowExplorerPage.xaml.cs [hunk 1]
```diff
@@ -839,20 +839,25 @@ private void SetControlsGridView()
...
-        private void RefreshControlsGrid()
+        private async void RefreshControlsGrid()
         {
-            if (WindowsComboBox.SelectedValue != null && mWindowExplorerDriver != null)
+            if (mWindowExplorerDriver != null)
             {
-                List<ElementInfo> list = mWindowExplorerDriver.GetVisibleControls(CheckedFilteringCreteriaList.Select(x => x.ElementType).ToList());
-                                
+                List<ElementInfo> list = await Task.Run(() => mWindowExplorerDriver.GetVisibleControls(CheckedFilteringCreteriaList.Select(x => x.ElementType).ToList()));
                 // Convert to obserable for the grid
```
Comment (MeniKadosh1) [inline Ginger/Ginger/WindowExplorer/Common/WindowExplorerPage.xaml.cs]: why the check for "WindowsComboBox.SelectedValue != null " was removed?

Comment (EliranBarDavid) [reply Ginger/Ginger/WindowExplorer/Common/WindowExplorerPage.xaml.cs]: Because its impossible to interact with a UI element in async function

Comment (MeniKadosh1) [reply Ginger/Ginger/WindowExplorer/Common/WindowExplorerPage.xaml.cs]: but it was added from some reason so we can't just avoid and remove this check.
so do it by storing the WindowsComboBox.SelectedValue on global field and update it on WindowsComboBox.SelectionChanged event and then check global field is null in IF

Comment (EliranBarDavid) [reply Ginger/Ginger/WindowExplorer/Common/WindowExplorerPage.xaml.cs]: Corrected

**Comment by EliranBarDavid** [comment]
> Code looks good but did you tested it for Java and Windows (grid/tree/LiveSpy)?

yes, It tested.

**Pull request merged**